crates/telio-traversal/src/session_keeper.rs

```diff
@@ -240,6 +241,19 @@ impl SessionKeeperTrait for SessionKeeper {
 
         Ok(())
     }
+
+    async fn get_interval(&self, key: &PublicKey) -> Option<u32> {
```

Comment:
Speaking about `SessionKeeper` - do we need it at all at this point? Since `Batcher` is doing everything now, it's not being used IMO. References show just one reference and it's inside a unit-test :)

Reply:
SK is used for both batched and nonbatched peers